machine.go

```diff
@@ -534,17 +534,17 @@ func (m *Machine) startup(command string, extracontent [][2]string) (int, error)
 		w.WriteDirectory("/lib", 0755)
 	}
 
-	prefix := ""
-	if mergedUsrSystem() {
-		prefix = "/usr"
-	}
-
 	// search for busybox; in some distros it's located under /sbin
 	busybox, err := exec.LookPath("busybox")
 	if err != nil {
 		return -1, err
 	}
-	w.CopyFileTo(busybox, prefix + "/bin/busybox")
+
+	if mergedUsrSystem() {
+		w.CopyFileTo(busybox, "/usr/bin/busybox")
+	} else {
+		w.CopyFileTo(busybox, "/bin/busybox")
+	}
 
```

Comment:
This is perfectly valid as-is, although it could be made simpler. Since we always have `/bin/` (as symlink in the merged case) we could use a single `w.CopyFileTo(busybox, "/bin/busybox")`

Reply:
actually this patch can be dropped since it doesnt add anything; but agree you could probably write a separate cleanup patch to not use `prefix` and just dump everything under `/bin`

Reply:
This is copying into a cpio archive; not a filesystem; The suggestion from @evelikov doesn't seem safe without validating the kernel cpio follows previously unpacked symlinks in that way but even if it does it would be somewhat unexpected so i don't think that's a good suggestion